codec-http/src/main/java/io/netty/handler/codec/http/websocketx/ContinuationWebSocketFrame.java

```diff
@@ -93,12 +93,22 @@ private static ByteBuf fromText(String text) {
 
     @Override
     public ContinuationWebSocketFrame copy() {
-        return new ContinuationWebSocketFrame(isFinalFragment(), rsv(), content().copy());
+        return (ContinuationWebSocketFrame) super.copy();
     }
 
     @Override
     public ContinuationWebSocketFrame duplicate() {
-        return new ContinuationWebSocketFrame(isFinalFragment(), rsv(), content().duplicate());
+        return (ContinuationWebSocketFrame) super.duplicate();
+    }
+
+    @Override
+    public ContinuationWebSocketFrame rduplicate() {
+        return (ContinuationWebSocketFrame) super.rduplicate();
+    }
+
+    @Override
+    protected ContinuationWebSocketFrame copy(ByteBuf content) {
```

Comment:
consider renaming `copy`. `copy` nomenclature is overloaded with `ByteBuf`'s copy.